code/__HELPERS/_logging.dm

```diff
@@ -26,99 +29,121 @@
 
 #ifdef UNIT_TESTS
 /proc/log_test(text)
-	WRITE_FILE(GLOB.test_log, "\[[time_stamp()]]: [text]")
+	WRITE_LOG(GLOB.test_log, text)
 	SEND_TEXT(world.log, text)
 #endif
 
+
+/* Items with ADMIN prefixed are stripped from public logs. */
```

Comment:
This is incorrect, should be `ADMINPRIVATE` prefix

Reply:
yeah, but according to MSO he's been doing both? @MrStonedOne

Reply:
if he is doing both there's literally no reason to keep the adminprivate tag, so if he does strip ADMIN just remove the adminprivate prefix and proc and update to admin

Reply:
Yeah it's both ADMIN and ADMINPRIVATE. I just kept the comment as ADMIN as technically ADMINPRIVATE can be considered prefixed with ADMIN.

Reply:
You changed it back in the .rs? (Context being mso saying he only filters adminprivate)